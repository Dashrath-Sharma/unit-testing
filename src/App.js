import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <h1>Unit-Testing</h1>
        <p>Snapshot Testing will be of App Component</p>
      </header>
    </div>
  );
}

export default App;