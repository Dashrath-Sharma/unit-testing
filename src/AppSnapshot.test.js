import App from "./App";
import { cleanup, render } from "@testing-library/react";

afterEach(cleanup);

it('App Snapshot Test', async () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot();
});