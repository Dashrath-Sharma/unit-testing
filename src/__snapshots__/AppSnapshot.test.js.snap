// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot Test 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <h1>
        Unit-Testing
      </h1>
      <p>
        Snapshot Testing will be of App Component
      </p>
    </header>
  </div>
</DocumentFragment>
`;
